get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = 'Bearer ' + new Buffer(AppConstants.HIBP_API_TOKEN);
```

Comment:
Only because I love you both:

> **/Users/pdehaan/dev/github/mozilla/blurts-server/get_hashsets.js**
>  11:31   **error**    `new Buffer()` was deprecated since [Node] v6. Use `Buffer.alloc()` or `Buffer.from()` (use 'https://www.npmjs.com/package/safe-buffer' for '<4.5.0') instead  node/no-deprecated-api

Reply:
Lol, nice, guess I was wrong about "less important". 👍🏼

Reply:
`Buffer` is only left-over from when I was base64-encoding the token. It's not needed anymore.

Reply:
Changed to template string with substitution like the rest of the constants.